src/main/java/frc/robot/subsystems/Storage.java

```diff
@@ -50,29 +43,30 @@ private Storage() {
 
     mTopRoller.setNeutralMode(NeutralMode.Brake);
     mBottomRoller.setNeutralMode(NeutralMode.Brake);
-
-    mIntakeSensor = new DigitalInput(INTAKE_SENSOR_PORT);
   }
 
-  public void init() {}
+  int getEncoder(){
+    return -mBottomRoller.getSelectedSensorPosition();
```

Comment:
Add a comment to clearly state that the encoder returns negative values for intake motion.

Reply:
Fixed